modules/serverless-configuring-burst-qps-for-net-kourier.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/kube-burst-qps-net-kourier.adoc
+
+:_content-type: REFERENCE
+[id="serverless-configuring-burst-qps-for-net-kourier_{context}"]
+= Configuring burst and QPS values for net-kourier
+
+The queries per second (QPS) value determines the number of client requests or API calls that are sent to the API server.
+
+The burst value determines how many requests from the client can be stored for processing. Requests exceeding this buffer will be dropped. This is helpful for controllers that are bursty in nature and do not spread their requests uniformly in time.
+
+When the `net-kourier-controller` restarts, it parses all `ingress` resources deployed on the cluster, which leads to a significant number of API calls. Consequently, it takes a long time for `net-kourier-controller` to start.
+
+You can adjust the QPS and burst values for `net-kourier-controller` in the KnativeServing CR:
+
+.KnativeServing CR example
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  workloads:
+  - name: net-kourier-controller
+    env:
+    - container: controller
+      envVars:
+      - name: KUBE_API_BURST
+        value: "200" <1>
+      - name: KUBE_API_QPS
+        value: "200" <2>
+----
++
```

Comment:
You can see at the bottom of the [preview](https://62138--docspreview.netlify.app/openshift-serverless/latest/knative-serving/kube-burst-qps-net-kourier.html#serverless-configuring-burst-qps-for-net-kourier_kube-burst-qps-net-kourier) that the callouts are not rendering properly. I tested it locally and removing the `+` should fix the issue
```suggestion

```